editor/edit-post/modes/visual-editor/inserter.js

```diff
@@ -67,6 +67,7 @@ export class VisualEditorInserter extends Component {
 						className="editor-inserter__block"
 						onClick={ () => this.insertBlock( block.name ) }
 						label={ sprintf( __( 'Insert %s' ), block.title ) }
+						disabled={ block.useOnce }
```

Comment:
What if the block hasn't been used yet in the post? Should the button not be disabled in these cases?

Reply:
You're right. I forgot this scenario but I did a new commit and I guess now is correct.